objc/core/A0LockReact.m

```diff
@@ -140,16 +141,38 @@ - (void)showWithOptions:(NSDictionary *)options callback:(A0LockCallback)callbac
     self.shown = YES;
 }
 
+- (void)getDelegationToken:(NSDictionary *)options callback:(A0LockCallback)callback {
+    if (!self.lock) {
+        callback(@[@"Please configure Lock before using it"]);
+        return;
+    }
+    A0APIClient *client = [self.lock apiClient];
+
+    void(^success)(NSDictionary *) = ^(NSDictionary *credentials) {
+        callback(@[[NSNull null], credentials]);
+    };
+    void(^failure)(NSError *) = ^(NSError *error) {
+        callback(@[[error localizedDescription]]);
+    };
+
+    A0AuthParameters *parameters = [A0AuthParameters newWithDictionary:@{
```

Comment:
I'll be adding some validation for those since it should not crash your app, just report an error. Let's add the validation for `api`, `id_token` and `scope` in your method.